Dockerfile

```diff
@@ -56,15 +54,13 @@ ENV GEM_PACKAGES="\
 
 ENV MOLECULE_EXTRAS="azure,docker,docs,ec2,gce,hetznercloud,linode,lxc,openstack,vagrant,windows"
 
+RUN apk add --update --no-cache ${BUILD_DEPS} ${PACKAGES}
+RUN gem install ${GEM_PACKAGES}
+RUN apk del --no-cache ${BUILD_DEPS}
```

Comment:
We should leave these in a single layer, I think the build deps will still contribute to the final image size because they are being stored in a separate layer.